.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
```

Comment:
What does labels do? Will this by default auto set issue's label?

Reply:
Yes, it auto sets a label.

Reply:
please remove default label.

Reply:
Done